### PR TITLE
Update the deps and manifests for release-1.3

### PR DESCRIPTION
**Pull request by houshengbo**

### CODE REVIEW

**Comment by kvmware** [comment]
/hold
There were some premature cuts that happened. I will updated you all when the branches are properly updated.

File: go.mod [hunk 1]
```diff
@@ -22,8 +22,8 @@ require (
 	k8s.io/code-generator v0.22.5
 	knative.dev/caching v0.0.0-20220302153644-5a6403ef2de5
 	knative.dev/eventing v0.30.0
-	knative.dev/hack v0.0.0-20220224013837-e1785985d364
-	knative.dev/pkg v0.0.0-20220301181942-2fdd5f232e77
+	knative.dev/hack v0.0.0-20220302134343-0687d6e8cd79
```
Comment (kvmware) [inline go.mod:25]: the hash for the 1.3 release for hack should be [e178598](https://github.com/knative/hack/commit/e1785985d364ff2ac43a33360a26c2027a02b1cf)

File: go.mod [hunk 2]
```diff
@@ -22,8 +22,8 @@ require (
...
+	knative.dev/pkg v0.0.0-20220302134643-d2cdc682d974
```
Comment (kvmware) [inline go.mod:26]: hash for pkg for the 1.3 release is [2fdd5f2](https://github.com/knative/pkg/commit/2fdd5f232e77efb2afd214ddc89e0b341508a700)

**Comment by kvmware** [comment]
The 1.3 release branches  for all repos should be in a good state now with the exception of `eventing-rabbitmq`.

File: go.mod (hunk 2 above)
Comment (houshengbo) [reply go.mod]: This is automatically generated by running the script: ./hack/update-deps.sh --upgrade --release v0.30

File: go.mod (hunk 1 above)
Comment (houshengbo) [reply go.mod]: This is automatically generated by running the script: ./hack/update-deps.sh --upgrade --release v0.30, so it should be fine.

**Pull request merged**